test/modular/Mocha6.js

```diff
@@ -40,12 +40,13 @@ export class Mocha6 extends Mocha {
 
   importFiles() {
     var promiseImports = this.files.map((file) => {
-      file = './' + file.replace(/\\/g, '/');;
-      this.suite.emit('pre-require', global, file, this);
+      file = './' + file.replace(/\\/g, '/');
+      // TODO(arv): Why do we need to use global here?
+      this.suite.emit('pre-require', Reflect.global, file, this);
```

Comment:
I just don't like us relying on the global object unless we have to.

The problem I encountered was that global was undefined (as expected).